src/Composer/Util/RemoteFilesystem.php

```
@@ -775,4 +823,113 @@ private function validateCaFile($filename)
 
         return (bool) openssl_x509_parse($contents);
     }
+
+    private function getCertificateCnAndFp($url, $options)
```

Yeah, I can make them fail hard on PHP >= 5.6 just in case.